docs/pages/ktable.vue

```diff
@@ -0,0 +1,156 @@
+<template>
+
+  <DocsPageTemplate apiDocs>
+    <DocsPageSection title="Overview" anchor="#overview">
+      <p>
+        The <code>KTable</code> component is an accessible and customizable table component designed to handle a variety of data presentation needs. The component is suitable for both simple and complex data tables. It offers:
+        <ul>
+          <li>Built-in sorting as well as integration with externally sorted data</li>
```

Comment:
I know Alex previously commented here, but I am actually not totally sure what this mean. I think if the distinction here is sorting without an API call, or sorting with an API call, it would be clearer to say that. Built-in sorting more or less makes sense to me, but the "externally sorted data" is not super clear to me.

Reply:
Hi @marcellamaki, thank you.

I suggested this phrasing. If it's not very clear, we should improve it, however I wouldn't recommend using the language of API calls here.

> if the distinction here is sorting without an API call, or sorting with an API call

The thing is that this is not the distinction. Rather it simply is - _does the table do sorting or does it receive sorted data from outside_? Here we want to say that it is designed flexibly enough to work well in both modes. When receiving sorted data from outside, it's not necessarily from waiting on an API call, it could actually be from frontend as well (won't go into detail much, but can elaborate if needed). Relatedly, KDS components as such don't really need to (and imho better don't) have information about all the details of all contexts they may be used in - that's intentional separation of concerns.

That's to provide context, not to dismiss. I think your commend shows we need to improve it. Taking the use-cases I mentioned into account, can you think of better way to communicate it? Perhaps we could just say it very simply "Offers built-in sorting by default" and "Integrates with already sorted data"/"Can receive already sorted data"? Or any other ideas?

Reply:
> "Offers built-in sorting by default" and "Integrates with already sorted data"

this sounds great to me! I know it isn't really that different, but for some reason, I find that easier to understand. :)

Reply:
Well apparently it is - seems the simpler the better :) Thanks

Reply:
@BabyElias could we make this tweak here please? Later on as we continue building the documentation, either during GSOC or after, this will set up the language nicely.

Reply:
Yes, sure!